### PR TITLE
Adds a new emergency shuttle: The JungleStation Exploration Shuttle

### PR DESCRIPTION
**Pull request by Jolly-66**
## About The Pull Request
Title.

## Why It's Good For The Game
It doesn't hurt to have more emergency shuttles, esp those with a little more of a "homey" feel.

While working on JungleStation itself, an area of the station resembles a shuttle. So, I decided to add it as its own shuttle as a nice little nod.

Its also purchasable by the RD, because it makes sense if the science team wants to ~~unwillingly~~ take the crew to go somewhere for a "nice" vacation. The shuttle costs 15,000 credits but I'm willing to work with the price.

~~It also kidnaps whatever crew is left over and takes them to deep space so we can find this damned planet~~

<details>
<summary> Image for the curious </summary>

image

</details>

## Changelog

:cl:
add: A new explorative emergency shuttle is here! The JungleStation Exploration shuttle can now be purchased by captains and Research Directors.
/:cl:

### CODE REVIEW

**Comment by Time-Green** [comment]
where's the jungle???.??>?

File: config/unbuyableshuttles.txt [hunk 1]
```diff
@@ -33,3 +33,4 @@
 #_maps/shuttles/emergency_tranquility.dmm
 #_maps/shuttles/emergency_venture.dmm
 #_maps/shuttles/emergency_wabbajack.dmm
+#_maps/shuttles/emergency_jungle_exploration.dmm
```
Comment (Mothblocks) [inline config/unbuyableshuttles.txt:36]: abc order

**Comment by SomeoneYouProbablyKnow** [comment]
This is way too big without a central gimmick go tie it all together. 
I get the funny planet thing but the shuttle itself is not interesting

**Comment by atlasle** [comment]
is junglestation still even being developped or....

**Comment by Jolly-66** [comment]
I'll bring this back whenever I get around to finishing JungleStation itself.

**Pull request closed without merging**